src/style.css

```diff
@@ -0,0 +1,124 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 11px;
+  line-height: 13px;
+  font-weight: 400;
+}
+
+body {
+  margin: 0;
+  --color-button: #00acdc;
+}
+
+h2 {
+  margin: 0;
+}
+
+/* #product-card */
+
+.page-content__card {
+  max-width: 200px;
+}
+
+.product-card {
+  padding: 29px 14px 14px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__container {
+  text-align: center;
+}
+
+.product-card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 0 42px;
+}
+
+.product-card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+  margin-bottom: 6px;
+}
+
+.product-card__subtitle {
+  margin-bottom: 15px;
+}
+
+/* #rating */
+
+.product-card__rating {
+  margin-bottom: 23px;
+}
+
+.rating {
+  display: flex;
+  justify-content: space-between;
+}
+
+.rating__stars {
+  display: flex;
+}
+
+.rating__star {
+  height: 15px;
+  width: 15px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.rating__star_active {
+  background-image: url(images/star-active.svg);
+}
+
+.rating__text {
+  padding-top: 1px;
+}
+
+/* #endrating */
+
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 21px;
+}
+
+.product-card__text {
+  padding-top: 2px;
+}
+
+.product-card__amount {
+  font-size: 16px;
+  font-weight: 700;
+}
+
+/* #button */
+
+.product-card__link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: var(--color-button);
+  border: 1px solid var(--color-button);
+  border-radius: 5px;
+  width: 170px;
```

Comment:
you don't need to set up fixed width: 170px; it is wrong approuch to set up sizes for element here

please set up indents from the left and right sides of your element in 15px; according to the task conditions
  margin-left: 15px;
  margin-right: 15px;

you have alredy set up display: flex; for .product-card__link  so now this block is not inline element but block type element so it spreads its width automaticaly 

you can also define the common css class for all elements on your card in indent 15px and to apply to all of them at once